packages/ubuntu_desktop_installer/integration_test/ubuntu_desktop_installer_test.dart

```diff
@@ -0,0 +1,513 @@
+import 'dart:io';
+
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_spinbox/flutter_spinbox.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:path/path.dart' as p;
+import 'package:subiquity_client/subiquity_client.dart';
+import 'package:ubuntu_desktop_installer/l10n.dart';
+import 'package:ubuntu_desktop_installer/main.dart' as app;
+import 'package:ubuntu_desktop_installer/pages.dart';
+import 'package:ubuntu_desktop_installer/pages/connect_to_internet/connect_model.dart';
+import 'package:ubuntu_desktop_installer/pages/updates_other_software/updates_other_software_model.dart';
+import 'package:ubuntu_desktop_installer/routes.dart';
+import 'package:ubuntu_test/utils.dart';
+import 'package:ubuntu_wizard/utils.dart';
+import 'package:ubuntu_wizard/widgets.dart';
+import 'package:yaml/yaml.dart';
+
+import '../test/widget_tester_extensions.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  setUp(() async => await cleanUpSubiquity());
+
+  testWidgets('minimal installation', (tester) async {
+    const language = 'Français';
+    const locale = 'fr_FR.UTF-8';
+    const timezone = 'Europe/Paris';
+    const keyboardLayout = KeyboardSetting(layout: 'Français');
+    const keyboardSetting = KeyboardSetting(layout: 'fr');
+    const identity = IdentityData(
+      realname: 'User',
+      hostname: 'ubuntu',
+      username: 'user',
+    );
+
+    app.main(<String>[]);
+    await waitForSubiquityServer();
+    await tester.pumpAndSettle();
+
+    await testWelcomePage(tester, language: language);
+    await tester.pumpAndSettle();
+
+    // https://github.com/canonical/ubuntu-desktop-installer/issues/373
+    // await testTryOrInstallPage(tester, option: Option.installUbuntu);
```

Comment:
Do you intend to revisit those two lines later? #373 is resolved by now. The question holds for both test cases.

Reply:
The task in #373 was to (temporarily) comment out or hide all non-functional UI parts before the release. At that point, the entire "Try or install" page was commented out. Perhaps the comment is now a bit misleading because #373 itself is closed, but the same comment is already sprinkled all around the codebase... :( I'd like to keep this regardless so it's easy to find all occurrences. Sorry for the confusion.